### PR TITLE
[uplift_beta] Fix a typo which leads to an error

### PR DESCRIPTION
**Pull request by calixteman**

### CODE REVIEW

File: runauto_nag_daily.sh [hunk 1]
```diff
@@ -109,13 +109,13 @@ python -m auto_nag.scripts.component --frequency daily
 # Try to detect potential wrong bug types using bugbug
 python -m auto_nag.scripts.defectenhancementtask
 
+# Try to detect potential missing Has STR using bugbug
+python -m auto_nag.scripts.stepstoreproduce
+
```
Comment (jcristau) [inline runauto_nag_daily.sh]: Is this change related?

**Comment by marco-c** [comment]
Oh, we should setup mergify like our other repos, or we can't r+ with nits.
